spark/src/main/scala/io/delta/sharing/spark/RemoteDeltaFileIndex.scala

```diff
@@ -179,21 +181,8 @@ private[sharing] case class RemoteDeltaBatchFileIndex(
   override def listFiles(
     partitionFilters: Seq[Expression],
     dataFilters: Seq[Expression]): Seq[PartitionDirectory] = {
-    // TODO(lin.zhou): Actually refresh the presigned url in the cache instead of just using
-    // getIdToUrlMap
-    CachedTableManager.INSTANCE
-      .register(params.path.toString, getIdToUrlMap, Seq(new WeakReference(this)), () => {
-        getIdToUrlMap
-      })
-
     // We ignore partition filters for list files, since the delta sharing server already
```

Comment:
We cannot ignore partition filters here. For streaming cases, this is okay as Spark doesn't support partition filter push down for streaming sources yet. But CDF batch queries, we need to apply the partition filters.

Reply:
But `RemoteDeltaBatchFileIndex` is used for the delta sharing streaming source? 
Do you mean I need to apply the partition filters in `RemoteDeltaCDFFileIndexBase.listFiles`?